ui/src/app/pages/admin/admin-banner.tsx

```diff
@@ -33,9 +34,10 @@ const validators = {
   bannerHeadline: {...required, ...notTooLong(200)},
   readMoreLink: {...notTooLong(200)},
 };
-export class AdminBanner extends React.Component<{}, AdminBannerState> {
+export class AdminBanner extends React.Component<WithSpinnerOverlayProps, AdminBannerState> {
   constructor(props) {
     super(props);
+    this.props.hideSpinner();
```

Comment:
In constructor bc routing spinner isn't about 'is this rendered' it's about 'do we know what page we're showing yet'

Reply:
Seems like this could have undesired consequences. I would expect this to either go in `componentDidMount()`, or not be necessary at all. Calling it in the constructor I think should be equivalent to not having a spinner - no?